ui/src/alerts/components/AlertsTable.js

```diff
@@ -82,9 +86,9 @@ const AlertsTable = React.createClass({
             </thead>
             <tbody>
               {
-                alerts.map(({name, time, value, host, level}) => {
+                alerts.map(({name, level, time, value, host}) => {
```

Comment:
Although object literals are orderless, I do see what you're doing here. You're going for consistency in which the values are used in the line below. If that's the case, you may also wish to switch value and host as well, just for consistency's sake.